zerver/views/user_settings.py

```diff
@@ -132,6 +132,7 @@ def update_display_settings_backend(
         twenty_four_hour_time: Optional[bool]=REQ(validator=check_bool, default=None),
```

Comment:
`zerver/views/__pycache__/pointer.cpython-34.pyc.140398671546048` should not be checked in.